python/json_util.py

```diff
@@ -0,0 +1,41 @@
+import mgp
+import json
+import urllib.request
+
+
+@mgp.read_proc
+def load_from_path(ctx: mgp.ProcCtx, path: str) -> mgp.Record(objects=mgp.List[object]):
+    """
+    Procedure to load JSON from a local file. 
+
+    Parameters
+    ----------
+    path : str
+        Path to the JSON that is being loaded.
+    """
+    with open(path) as json_file:
```

Comment:
Check whether the path exists before opening. Also, I would recommend using `pathlib`, it comes from standard Python.
`from pathlib import Path`

Reply:
Added that in the new commit, not sure if that's what you were thinking about.